### ISSUE TITLE
what about bundled apk's?

### ISSUE THREAD

**Issue by Lyceris-chan** [tytydraco/appdb#1]
trebleshot FOSS gives split APK's when transfering apps how is this script gonna act on them?

**Comment by tytydraco** [comment]
Not quite sure what you mean. For me I pull all the APKs including those from a split apk. Then you can select them all with SAI and install them.

**Comment by Lyceris-chan** [comment]
yeah I wasn't sure if the apk's would be split as well that was kinda the question lmao
